tests/roots/test-ext-viewcode-find-package/main_package/subpackage/__init__.py

```diff
@@ -1,3 +1,3 @@
-from main_package.subpackage._subpackage2 import submodule  # NOQA
+from main_package.subpackage._subpackage2 import submodule  # noqa: F401
 
-__all__ = ["submodule"]
\ No newline at end of file
+__all__ = ["submodule"]
```

Comment:
do you still need noqa: F401 fi you're explicitly exporting names with `__all__ = [...]`? I would hope flake8 would be smart enough to recognise that

Reply:
No I don't think so. I'll remove that line